If not using Azure Shell, mention need to login

andynava opened the pull request:
Mention that user needs to log into account if they are running the CLI locally.
"Quick start" -> "Quickstart" for consistency.

azurecla commented:
Hi __@yossibanai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@yossibanai__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

BryanLa commented:
Working with @yossibanai and @curtand to get the changes redirected to the private repo.

BryanLa commented:
Closing for now, as there have been several updates to article since submitting. We will continue to work with Yossi (and the entire AAD engineering team) to redirect changes via private repo.  We can reopen if necessary.

andynava committed:
If not using Azure Shell, mention need to login

Mention that user needs to log into account if they are running the CLI locally.
"Quick start" -> "Quickstart" for consistency.

JasonWHowell commented on articles/postgresql/quickstart-create-server-database-azure-cli.md:
```diff
@@ -20,6 +20,11 @@ If you don't have an Azure subscription, create a [free](https://azure.microsoft
 
 If you choose to install and use the CLI locally, this topic requires that you are running the Azure CLI version 2.0 or later. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI 2.0]( /cli/azure/install-azure-cli). 
 
+If you are running the CLI locally, you will need to log into your account using the [az login](/cli/azure/authenticate-azure-cli?view=interactive-log-in) command.
```
Have to use present tense here. "you need to log in to your account..."

andynava committed:
Use present tense
